src/axom/sidre/core/Group.hpp

```diff
@@ -177,12 +176,14 @@ class Group
    */
   std::string getPathName() const
   {
-    if (getPath().length() < 1)
+    const auto p = getPath();
```

Comment:
Efficiency optimization here -- ``getPath`` is only called at most once per ``getPathName()``, rather than twice (in the typical case).